api/data_explorer/dataset_faceted_search.py

```diff
@@ -48,7 +48,7 @@ def get_facets():
         else:
             # Assume numeric type.
             # TODO: Handle other types.
-            facets[field_name] = HistogramFacet(field=field_name, interval=7)
+            facets[field_name] = HistogramFacet(field=field_name, interval=10)
```

Comment:
Since this is a quick hack, could we add a TODO here to more intelligently assign bucket intervals depending on the range of the data that we're looking at? I'm not sure exactly what that would look like, maybe it would go in the config.

Reply:
Added TODO. API server should figure it out on the fly. I don't want it go in configs, it would make configs more complicated. If people really wanted to specify custom ranges, we perhaps make it an optional config, though I would prefer not to.

For certain fields, we should hardcode ranges. For example, for age, "65-over" is common. Or for LDL cholesterol, there are standard ranges.